### PR TITLE
Release/1.15.0

### PR DESCRIPTION
**Pull request by skorper**
### Added
- Added support for Harmony subsetting

### CODE REVIEW

File: subscriber/podaac_data_downloader.py [hunk 1]
```diff
@@ -244,7 +284,7 @@ def run(args=None):
                       results['items']]
     downloads_metadata = [[u['URL'] for u in r['umm']['RelatedUrls'] if u['Type'] == "EXTENDED METADATA"] for r in
                           results['items']]
-    checksums = pa.extract_checksums(results)
+    checksums = pa.extract_checksums(results['items'])
```
Comment (hkryeung) [inline subscriber/podaac_data_downloader.py:287]: I'd use `results.get('items', [])` to either return an empty list if somehow `items` key doesn't exist; but if that check is happening before / guaranteed to be returned then it works!

File: subscriber/podaac_data_subscriber.py [hunk 2]
```diff
@@ -236,20 +236,85 @@ def run(args=None):
...
+    if len(granules) > 0 and not args.dry_run:
+        if not failure_cnt > 0:
+            with open(f'{data_path}/.update__{args.collection}', 'w') as f:
+                f.write(datetime.utcnow().strftime("%Y-%m-%dT%H:%M:%SZ"))
+
+    if success_cnt > 0:
+        try:
+            pa.create_citation_file(short_name, provider, data_path, token, args.verbose)
+        except:
+            logging.debug('Error generating citation', exc_info=True)
```
Comment (hkryeung) [inline subscriber/podaac_data_subscriber.py:296]: should this be `logging.error` instead?

**Review by hkryeung — APPROVED** [review]
Added in some minor comments but overall code looks good!

File: subscriber/podaac_data_downloader.py (hunk 1 above)
Comment (skorper) [reply subscriber/podaac_data_downloader.py:287]: Good point Randy -- luckily CMR always returns `items` even when the query matches 0 results

Comment (hkryeung) [reply subscriber/podaac_data_downloader.py:287]: Roger that; consider this resolved!

**Pull request merged**